src/Assets/AllSVG.jsx

```diff
@@ -0,0 +1,189 @@
+function SunIcon(props) {
```

Comment:
Any reason for building custom icons rather than using prebuilt libs?

Reply:
Nope, I have been using custom icons so just continuing them